docs/marketing/play-store/store-listing.md

```
@@ -0,0 +1,167 @@
+# Disciplefy - Play Store Listing Content
+
+## App Name
+**Disciplefy - Bible Study**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)